doc/eID/eid-flow-paula.md

```diff
@@ -21,38 +21,44 @@ sequenceDiagram
-    backend ->> backend: generate QR Code [eIDClientURL]
-    backend -->> widget: return widget
+    backend -->> eService: return <br>[tcTokenURL]
+    eService ->> widget: integrate script which creates iframe <br>[tcTokenURL]
+    widget ->> backend: (iframe) get widget page
+    backend -->> widget: return widget page
+    widget ->> widget: generate <br>[widgetSessionId, widgetSessionSecret]
+    widget ->> backend: open SSE channel <br>[widgetSessionId]
+    backend -->> widget: return
+    widget ->> widget: generate QR Code <br>[eIDClientURL incl. tcTokenURL, widgetSessionId, widgetSessionSecret]
     widget -->> user: display widget
     user ->> smartphone: open camera
     smartphone ->> widget: scan QR Code
-    widget -->> smartphone: return [eIDClientURL]
-    smartphone ->> app: open [tcTokenURL]
+    widget -->> smartphone: return <br>[eIDClientURL]
+    smartphone ->> app: open via eIDClientURL <br>[tcTokenURL, widgetSessionId, widgetSessionSecret]
     app ->> backend: get tcToken from tcTokenURL
-    backend -->> app: return [tcToken]
+    backend -->> app: return <br>[tcToken]
     app ->> server: establish connection
-    server -->> app: return [list of requested data]
+    server -->> app: return <br>[list of requested data]
     app -->> user: display requested data
     user ->> app: approve data request
     app ->> smartphone: access NFC reader
     smartphone ->> id: scan NFC chip
-    id -->> smartphone: return [encrypted identity data]
-    app ->> server: send [encrypted identity data]
-    server ->> server: decrypt [encrypted identity data] 
+    id -->> smartphone: return <br>[encrypted identity data]
+    smartphone -->> app: return <br>[encrypted identity data]
+    app ->> server: send <br>[encrypted identity data]
+    server ->> server: decrypt <br>[encrypted identity data] 
     server -->> app: return success
-    app ->> backend: inform about success [sessionId]
-    backend ->> widget: inform about success (via SSE) [sessionId]
-    widget ->> browser: redirect to [refreshAddress]
+    app ->> app: encrypt refreshURL with widgetSessionSecret <br>[encrypted refreshURL]
+    app ->> backend: inform about success <br>[encrypted refreshURL, widgetSessionId]
+    backend ->> widget: inform about success (via SSE) <br>[encrypted refreshURL, widgetSessionId]
+    widget ->> widget: decrypt refreshURL with widgetSessionSecret <br>[refreshURL]
+    widget ->> browser: redirect to <br>[refreshAddress]
     browser ->> eService: access success page
-    eService ->> backend: get identity data [sessionId]
-    backend ->> server: get identity data [sessionId]
-    server -->> backend: return [decrypted identity data]
-    backend -->> eService: return [decrypted identity data]
+    eService ->> backend: get identity data <br>[sessionId]
+    backend ->> server: get identity data <br>[sessionId]
+    server -->> backend: return <br>[decrypted identity data]
```

Comment:
Actually, we don't need the keyword `decrypted` here. It's more descriptive, but then we could add it at `refresh address` as well? For me both is fine.

Reply:
I agree. I will remove the "decrypted" to keep it lean.